package.json

```diff
@@ -27,7 +27,7 @@
   ],
   "dependencies": {
     "@grpc/grpc-js": "^1.1.7",
-    "@nitric/api": "^0.13.0",
+    "@nitric/api": "rc-latest",
```

Comment:
Not sure if this should be set to the latest rc version.

Reply:
Made change to lock version to 0.14.0-rc.4